modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/network/disaster/LocalPartitionsEstimatedSizeMessage.java

```diff
@@ -27,7 +27,7 @@
  * A message for reading estimated number of rows for tables stored on this node.
  */
 @Transferable(DisasterRecoveryMessages.LOCAL_TABLE_PARTITION_STATE)
-public interface LocalTablePartitionStateMessage extends NetworkMessage {
+public interface LocalPartitionsEstimatedSizeMessage extends NetworkMessage {
```

Comment:
I'm not sure that we need such request and such logic. As far as I remember, it was added for compatibility mode in order to retrieve table partitions states in colocation mode. For more details please check `DisasterRecoveryManager#tableStateForZone` along with corresponding usages `DisasterRecoveryManager#localTablePartitionStates` and `DisasterRecoveryManager#globalTablePartitionStates`. I assume that we won't provide table partition states any longer only zone ones.